recipes/fisspy/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "fisspy" %}
+{% set version = "0.9.61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b80ba11af1916e06e83d411909c6c5a7fa7e5060b2f6fa2cad2f90f6a6a29869
+
+build:
+  noarch: python
+  number: 2
```

This should start at 0
```suggestion
  number: 0
```